internal/stats/stats_service.go

```diff
@@ -424,6 +439,43 @@ func (g gaugeOS) updateDescr(bt *binTextStats) {
 	bt.stats = append(bt.stats, makeTextStat(&g.openDecrCount, nfd))
 }
 
+type gaugeNearCache struct {
+	serviceName string
+	f           func() func(service string) NearCacheStatsGetter
+}
+
+func newGaugeNearCache(svc string, f func() func(service string) NearCacheStatsGetter) gaugeNearCache {
```

Comment:
Don't we need to send near cache stats as basic stats as well? Java sends in both formats as far as I see.

Reply:
Yes, but that's required only for 4.x. Since that's out of support, sending only the binary stats.

Reply:
ClientMetricsConfig class has the annotation `@since 4.0`. So is it added in 4.0? Why do we send basic stats in our clients anyway? 🤔 Good point to discuss.